static/app/views/releases/detail/overview/chart/utils.tsx

```diff
@@ -21,9 +21,16 @@ type ChartData = Record<string, Series>;
 
 const SESSIONS_CHART_PALETTE = CHART_PALETTE[3];
 
-export function getInterval(datetimeObj: DateTimeObject) {
+export function getInterval(datetimeObj: DateTimeObject, organization?: Organization) {
   const diffInMinutes = getDiffInMinutes(datetimeObj);
 
+  if (
+    organization?.features.includes('minute-resolution-sessions') &&
```

Comment:
Instead of requiring the organization as an argument, I think it would make more sense to have a boolean for enabling minute resolution sessions.